src/widget/city_building_ghost.c

```diff
@@ -4,6 +4,7 @@
 #include "building/count.h"
 #include "building/industry.h"
 #include "building/properties.h"
+#include "core/config.h"
 #include "city/buildings.h"
 #include "city/finance.h"
 #include "city/view.h"
```

Comment:
Alphabetical order please :)
```suggestion
#include "city/buildings.h"
#include "city/finance.h"
#include "city/view.h"
#include "core/config.h"
```